browser/brave_wallet/filecoin_keyring_unittest.cc

```diff
@@ -171,6 +167,28 @@ TEST(FilecoinKeyring, fil_private_key_public_key) {
   ASSERT_TRUE(std::all_of(zero_key.begin(), zero_key.end(),
                           [](int i) { return i == 0; }));
 }
-#endif
+
+TEST(FilecoinKeyring, SignTransaction) {
```

Comment:
they are coming from my tests. I checked it to work on real networks and generated tests with localy generated test addresses

Reply:
I suppose that makes sense because we're relying on an implementation specifically provided by filecoin - https://github.com/filecoin-project/bls-signatures

I did find the underlying library of this is [well tested](https://github.com/filecoin-project/bls-signatures/blob/master/tests/data.json) as well however, when looking at [the commit that added those test vectors](https://github.com/filecoin-project/bls-signatures/commit/496600164534e2521f893c241a5abe82219d945e) I realized those test vectors are building off hash-to-curve-v7 when that spec is [now at v14](https://datatracker.ietf.org/doc/draft-irtf-cfrg-hash-to-curve/).

All in all, I think this is probably fine as is because the underlying library does a great job of checking the implementation, but it does make me wonder if the underlying filecoin network needs to be updated. Since the hash to curve spec is behind and the BLS signatures [spec is expired](https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-bls-signature-04).

I don't think this should be a blocking change, but @gpestana or @alxdavids when you look at the rust code can you take a double check over these things?

Reply:
the `bls-signature` is just underlaying lib, we had sec review for it earlier and some tests could be found in `filecoin_keyring_unittest.cc`. Here I test the signature for a message, which is bls/secp over binary serialized message

Reply:
Yup, I double checked that and think this one is good. Resolving